Tiltfile

```
@@ -343,6 +343,23 @@ def deploy_observability():
         k8s_yaml(read_file("./.tiltbuild/yaml/prometheus.observability.yaml"), allow_duplicates = True)
         k8s_resource(workload = "prometheus-server", new_name = "prometheus", port_forwards = "9090", extra_pod_selectors = [{"app": "prometheus"}], labels = ["observability"])
 
+    if "cluster-api-visualizer" in settings.get("deploy_observability", []):
+        internal_kubeconfig = str(local("kind get kubeconfig --name ${KIND_CLUSTER_NAME:-capi-test} --internal", quiet=True))
+        k8s_yaml(helm(
+            "./hack/observability/cluster-api-visualizer/charts/cluster-api-visualizer",
```

```suggestion
            "./hack/observability/visualizer/charts/cluster-api-visualizer",
```
And also rename the folder